_rules/live-region-input-error-2045c3.md

```diff
@@ -0,0 +1,341 @@
+---
+id: 2045c3
+name: alert role or live region identify input error
+rule_type: atomic
+
+description: |
+  This rule checks that a container with `role=alert` or a live region are used to identify input errors.
+
+accessibility_requirements:
+  wcag-technique:ARIA19: # Using ARIA role=alert or Live Regions to Identify Errors
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+
+authors:
+  - Carlos Duarte
+  - João Vicente
+---
+
+## Applicability
+
+This rule applies to any [document](https://www.w3.org/TR/dom/#concept-document) where the [document element](https://www.w3.org/TR/dom/#document-element) is an HTML `html` element that:
+
+- has at least one HTML or SVG element that has one of the following [semantic roles](#semantic-role): `checkbox`, `combobox`, `listbox`, `menuitemcheckbox`, `menuitemradio`, `radio`, `searchbox`, `slider`, `spinbutton`, `switch` and `textbox` and for which [input errors](https://www.w3.org/TR/WCAG21/#dfn-input-error) are [automatically detected](#automatic-error-detection)
+- has at least one [descendant](https://dom.spec.whatwg.org/#concept-tree-descendant) that is an HTML element with either `role=alert` or `aria-live=assertive` attributes that is present in the [document](https://www.w3.org/TR/dom/#concept-document) at the time it [completes loading](https://www.w3.org/TR/html52/dom.html#dom-documentreadystate-complete).
```

Comment:
I think the formulation of "readiness" used in your other pull request is more clear: https://github.com/act-rules/act-rules.github.io/pull/960/files#diff-c66e018743316a225db096771fafa5f9R28